JSDemos/Demos/DataGrid/BatchEditing/description.md

```diff
@@ -1,5 +1,5 @@
 You can use batch edit mode to defer saving multiple changes. In this mode, users edit data one cell at a time. The changes are stored in a buffer and can be discarded before the **Save** button is clicked.
 
-To enable batch edit mode, assign *"batch"* to the **editing**.[mode](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/editing/#mode) property.
+To enable batch edit mode, set the [mode](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/editing/#mode) property to *"batch"* and enable the [editing](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/editing/#mode) object's [allowUpdating](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/editing/#allowUpdating), [allowAdding](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/editing/#allowAdding), and [allowDeleting](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/editing/#allowDeleting) properties to allow edit operations.
```

Comment:
This sentence looks too complex. How about making a list like here: https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/FormEditing/Vue/Light/

```suggestion
To enable the batch edit mode, set the [mode](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/editing/#mode) property to *"batch"* and enable the [editing](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/editing/#mode) object's [allowUpdating](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/editing/#allowUpdating), [allowAdding](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/editing/#allowAdding), and [allowDeleting](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/editing/#allowDeleting) properties to allow edit operations.
```